src/Label.js

```diff
@@ -43,7 +43,7 @@ const sizeVariant = variant({
 
 const Label = styled('span')`
   display: inline-block;
-  font-weight: 600;
+  font-weight: 400;
```

Comment:
Should we use a theme value here?

```suggestion
  font-weight: ${get('fontWeights.normal')};
```